libs/sdk-ui-dashboard/src/presentation/filterBar/attributeFilter/addAttributeFilter/AttributesDropdown.tsx

```diff
@@ -101,6 +101,24 @@ export function AttributesDropdown({
     const intl = useIntl();
     const [searchQuery, setSearchQuery] = useState("");
 
+    const [closeOnParentScroll, setCloseOnParentScroll] = useState(false);
+
+    // this effect is here just for preventing bad behavior and manage safe dropdown opening
+    // when adding this placeholder into filterBar it could change height of Dashboard header (not happening by default)
+    // changing height of sticky element will fire onScroll event (size of scroll area changed)
+    // KD or other app could listen to this event and call immediately global event to close all opened overlays
+    // so we prevent this by setting closeOnParentScroll true with some delay 0.5
+
+    useEffect(() => {
+        const timer = window.setTimeout(() => setCloseOnParentScroll(true), 500);
```

Comment:
I don't understand the comment, why is this timeout necessary?

Reply:
Use case: 

I have custom (DateFilter) its height is 50 px (default is 56)
But I do not have custom AttributesDropdown (height is 56) this operation will change height of sticky FilterBar, this will change height of dashboard scroll area, this will be catch by KD (window scroll) this will fire goodstrap global event and this will immediately close all open overlays that have   CloseOnParentScroll = true 

So result is that AttributesDropdown will be immediately closed after adding and user is not able select AttributeFilter to ADD.

Reply:
Fixed